### PR TITLE
Change native keyword to extern

### PR DESCRIPTION
**Pull request by Shan1024**
## Purpose
This PR updates the `native` keyword to `extern`.

### CODE REVIEW

**Comment by hasithaa** [comment]
@Shan1024 , Can you fix conflicts?

**Comment by hasithaa** [comment]
@hevayo @pahans @nadeeshaan @NipunaRanasinghe Can you check the impact of this change with IDEs. ?

**Comment by NipunaRanasinghe** [comment]
@hasithaa ack and will update the IDEA plugin accordingly

File: compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangParserListener.java [hunk 1]
```diff
@@ -55,7 +55,7 @@
  */
 public class BLangParserListener extends BallerinaParserBaseListener {
     private static final String KEYWORD_PUBLIC = "public";
-    private static final String KEYWORD_NATIVE = "native";
+    private static final String KEYWORD_NATIVE = "extern";
```
Comment (grainier) [inline compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangParserListener.java]: Shouldn't this be changed to `KEYWORD_EXTERN`? Just for the sake of consistency.

**Pull request merged**